dotenv/mod_test.ts

```diff
@@ -1,434 +1,153 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
-import {
-  assertEquals,
-  assertRejects,
-  assertThrows,
-} from "../testing/asserts.ts";
-import { config, configSync, MissingEnvVarsError, parse } from "./mod.ts";
+import { assertEquals, assertThrows } from "../testing/asserts.ts";
+import { parse, verify } from "./mod.ts";
 import * as path from "../path/mod.ts";
 
 const moduleDir = path.dirname(path.fromFileUrl(import.meta.url));
 const testdataDir = path.resolve(moduleDir, "testdata");
 
-const testOptions = {
-  path: path.join(testdataDir, "./.env"),
-  defaults: path.join(testdataDir, "./.env.defaults"),
-};
-
-Deno.test("parser", () => {
-  const testDotenv = Deno.readTextFileSync(
-    path.join(testdataDir, "./.env.test"),
+Deno.test("parse", () => {
```

Comment:
I think it makes sense to use the [test steps API](https://deno.land/manual@v1.19.2/testing#test-steps) here, because the assertions test independent behaviors of the function.

e.g.

```ts
Deno.test("parse", async (ctx) => {
  // ...
  await ctx.step("parses a basic variable", () => assertEquals(env.BASIC, "basic"));
  await ctx.step("skips empty lines", () => assertEquals(env.AFTER_EMPTY, "empty"));
  // etc...
});
```